spec/styles_spec.rb

```diff
@@ -71,6 +71,16 @@
         end
       end
 
+      it "must not use sentence case for title and container-title variables" do
+        style.descendants!.each do |node|
+          if node.matches?("text")
+            if node[:'text-case'] == 'sentence'
+              expect(node[:'variable']).not_to match(/^title|container-title$/)
+            end
+          end
+        end
+      end
+
```

Comment:
@adam3smith, the logic is not the most elegant, but RSpec is still mostly a mystery to me. It seems to work and doesn't seem to be particularly slow, so I guess it's okay.